src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/ApplicationRepository.cs

```diff
@@ -58,73 +59,116 @@ public ApplicationRepository(IOptions<AzureCosmosSettings> cosmosettings)
             _client.OpenAsync();
         }
 
+        private string AppIdToCosmosId(string appId)
+        {
+            string[] parts = appId.Split("/");
```

Comment:
Im assuming it has been successfully tested using " " vs '' . 
dotnetfiddle complained about it being a string and not a char.